### PR TITLE
FIX: Removes mutexes. Opens db only once

### PR DESCRIPTION
**Pull request by nhatcher**
Have a look at this!

### CODE REVIEW

File: server.go [hunk 1]
```diff
@@ -109,7 +109,7 @@ func mainHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func main() {
-	testDB()
+	initDatabase()
```
Comment (Carlos7979) [inline server.go:112]: Hey Nico! Very good change of name.

**Review by Carlos7979 — CHANGES_REQUESTED** [review]
Are you sure you are closing the database properly?

**Comment by nhatcher** [comment]
Hey I think it is. I guess that's the way it works when the app exits

**Review by Carlos7979 — APPROVED** [review]
ok, let's merge this!

**Pull request merged**